lua/plenary/profile.lua

```diff
@@ -0,0 +1,11 @@
+local profile = {}
+
+function profile.benchmark(f, iterations, ...)
+  local start_time = vim.loop.hrtime()
+  for _ = 1, iterations or 1E9 do
```

Comment:
I feel like `1E9` is a fair default, but just wanted to verify.

Reply:
I'd like to remove the default, and make `iterations` be the first variable.

So it looks like `profile.bencharm(1E9, my_func, arg1)`. It's confusing to pass the iterations after the function but not pass it to the function I think.